data/settings_mapping.json

```diff
@@ -305,6 +305,7 @@
           "text": "Misc",
           "row_span": [1,1,1],
           "settings": [
+			"no_collectible_hearts",
```

Comment:
This should probably go below damage multiplier, as it's similar to that kind of challenge, and the first few options here are pretty commonly used.

Reply:
I agree. That was only put there as a means of testing and I forgot to move them. Thank you for doing so (at least I think you did.)

Reply:
I'll take care of that.